week-4/B-objects-get-set/exercise-4.js

```diff
@@ -9,7 +9,8 @@ var dog = {
 
 // WRITE CODE BELOW THIS LINE
 
-
+dog.name = "Rex"
+dog.wantsToPlay = true

```

Comment:
Please make sure to always end a line of code on `;`. Again, just a minor detail.